### ISSUE TITLE
Customize home screen to display collections as sections

### ISSUE THREAD

**Issue by vortex91** [jellyfin/jellyfin-androidtv#4241]
### Describe the feature you'd like

Tow part request:  
Currently I use plugin called jellyfind auto collection to generate dynamic collection lists such as "trending" "sci-fi" etc... these are set as cron jobs and update via trakt IMDb etc..  Currently I have to navigate to collections to view these items.  It would be great if I could set several of these as sections in home screen to have Netflix like experiance. (not all collections but pick  a specific collection and display its items)

2nd part:
Currently the view of a specific collection in collections is limited to single line scroll horizontaly. It would be great to be able to change the view to ALL items for collections which have many items.

**Comment by vortex91** [comment]
home sections plugin does this for web UI.  I hope offical Jellyfin web incorporates it also.   Android tv should also be as flexible in picking up sections info from server instead of hard coded.

**Comment by nothing2obvi** [comment]
Yes, I think it would be great if Jellyfin Android TV could either mimic the home screen as seen on the web UI, or if that's not possible, create some kind of plugin that could inject custom sections into the client, similar to how the Streamyfin plugin does it: https://github.com/streamyfin/jellyfin-plugin-streamyfin

**Comment by nielsvanvelzen** [comment]
The app implement the "home section" API and only supports the features that are exposed with it. For addition to this functionality you should request them at features.jellyfin.org.
